### PR TITLE
Added KUBCONFIG to /etc/bash.bashrc

### PR DESCRIPTION
**Pull request by wulfthimm**
## Description

This sets the KUBECONFIG environment variable globally so it is automatically set after a SSH login.

## Related Issue(s)

Fixes https://github.com/gitpod-io/ops/issues/638

## How to test

Start a new VM and run `./dev/preview/ssh-vm.sh` to access the VM. `helm list` will show an empty list. Youi can also add a new repo and install any helm chart.

## Release Notes

```release-note
None
```

### CODE REVIEW

**Comment by roboquat** [comment]
@wulfthimm: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kylos101** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @kylos101` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: .werft/vm/vm.ts
Comment (wulfthimm) [inline .werft/vm/vm.ts:25]: This is a fix for removing all resources during a `clean-slate-deployment`.

**Comment by ArthurSens** [comment]
Are you linking the correct issue? 🤔

**Comment by wulfthimm** [comment]
> Are you linking the correct issue? thinking

Nope, https://github.com/gitpod-io/ops/issues/638

File: .werft/vm/manifests.ts
Comment (ArthurSens) [inline .werft/vm/manifests.ts:212]: Nice that we don't need to log as root to use kubectl anymore :)

**Review by ArthurSens — APPROVED** [review]
Tested and looks good!

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: eb0d63ecf9675c6453ed844b1d37098fdc88a6cc</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7974#pullrequestreview-870383120" title="Approved">ArthurSens</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kylos101** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @kylos101` in a comment when ready.

Associated issue: *#638*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7974#pullrequestreview-870383120" title="Approved">ArthurSens</a>*

Associated issue: *#638*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ArthurSens** [comment]
@wulfthimm, the build is failing and tide wont merge this until it is fixed 😬 

Could you make sure we are not breaking our CI before getting it merged?

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7974#pullrequestreview-870383120" title="Approved">ArthurSens</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kylos101** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @kylos101` in a comment when ready.

Associated issue: *#638*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by wulfthimm** [comment]
> @wulfthimm, the build is failing and tide wont merge this until it is fixed grimacing
> 
> Could you make sure we are not breaking our CI before getting it merged?

I removed the werft labels.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7974#pullrequestreview-870383120" title="Approved">ArthurSens</a>*

Associated issue: *#638*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
